src/dodal/log.py

```diff
@@ -127,25 +129,28 @@ def set_up_logging_handlers(
     dev_mode: bool = False,
     logging_path: Optional[Path] = None,
     file_handler_logging_level: Optional[str] = None,
+    logger=LOGGER,
 ) -> List[logging.Handler]:
     """Set up the default logging environment.
     Args:
         logging_level: The level of logs that should be saved to file/graylog. Defaults to INFO.
         dev_mode: True if in dev mode, will not log to graylog in dev. Defaults to False.
         logging_path: The location to store log files, if left as None then puts them in the default location.
     """
-    logging_level = logging_level if logging_level else "INFO"
+    logging_level = logging_level or "INFO"
+    if dev_mode is None:
+        dev_mode = True
```

Comment:
Should: Now that the `dev_mode` is not optional I think we can leave it up to the type checker to check this